terracotta/cmaps/__init__.py

```diff
@@ -36,7 +36,7 @@ def get_cmap(name: str) -> np.ndarray:
         raise ValueError(f'Unknown colormap {name}, must be one of {AVAILABLE_CMAPS}')
 
     with _get_cmap_data(name) as f:
-        cmap_data = np.load(f)
+        cmap_data = np.fromfile(f, dtype=np.uint8).reshape((255, 3))
```

Comment:
See comment above; could we give `.npy` files one more shot with the additions to `.gitattributes`?